docs/content/cli.md

```diff
@@ -618,6 +618,7 @@ opa exec <path> [<path> [...]] [flags]
       --log-timestamp-format string          set log timestamp format (OPA_LOG_TIMESTAMP_FORMAT environment variable)
       --set stringArray                      override config values on the command line (use commas to specify multiple values)
       --set-file stringArray                 override config values with files on the command line (use commas to specify multiple values)
+      --timeout duration                     set exec timeout (default unlimited)
```

Comment:
It would be good to say something about the expected time unit here, or format here. If I provide the value "10", what do I get? I saw you used 10s in the changelog example, but that's not likely to be where people look for that information.

Reply:
Good idea! I've added an example duration string in my latest changes.